.github/workflows/ci.yml

```diff
@@ -2,15 +2,15 @@ name: CI
 
 on:
   push:
-    branches:
-      - master
+    branches: [master]
```

Comment:
why did this need to change?

Reply:
its a cleaner way - i can revert to older version